### PR TITLE
Finished css-fonts exercise

### PR DESCRIPTION
**Pull request by sanfordhuynh**
css-font

### CODE REVIEW

**Comment by sanfordhuynh** [comment]
Added link tag to stylesheet

css-font

File: css-fonts/styles.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital@1&family=Poppins:wght@100&display=swap');
```
Comment (kurtyazdizadeh) [inline css-fonts/styles.css]: This is another way to import fonts into your page! However, you don't need to use the `@import` rule here if you are using the `<link>` tag in your HTML file (they are doing the same thing).

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Remove the `@import` in your CSS file since you are importing the font via the `<link>` tag in your HTML file

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**